lib/internal/modules/esm/worker.js

```diff
@@ -107,7 +106,7 @@ async function customizedModuleWorker(lock, syncCommPort, errorHandler) {
   if (hasInitializationError) {
     syncCommPort.postMessage(wrapMessage('error', initializationError));
   } else {
-    syncCommPort.postMessage(wrapMessage('success', { preloadScripts }), preloadScripts.map(({ port }) => port));
```

Comment:
@aduh95 @izaakschroeder would this need to be replaced with something from/for `initialize`, or does `initialize` take care of that itself?

Reply:
I would imagine we still want a success message to confirm the loader worker is up and running but I don't think it needs to send the scripts anymore? Initialize takes care of that part (though it runs on the worker thread vs globalPreload that ran on the main thread afaik).